types/node/index.d.ts

```diff
@@ -756,20 +761,23 @@ declare module "http" {
         write(str: string, encoding?: string, cb?: Function): boolean;
         write(str: string, encoding?: string, fd?: string): boolean;
 
-        writeContinue(): void;
-        writeHead(statusCode: number, reasonPhrase?: string, headers?: any): void;
-        writeHead(statusCode: number, headers?: any): void;
-        statusCode: number;
-        statusMessage: string;
+        addTrailers(headers: ServerResponseHeaders): void;
+        finished: boolean;
+        getHeader(name: string): number | string | string[] | undefined;
```

Comment:
Turns out dtslint gets *almost* all the way through definitely typed  before Travis kills it. Searching for errors turns up 1 error in `content-type`: 

```
Testing content-type
	Running: node /home/travis/build/DefinitelyTyped/DefinitelyTyped/node_modules/types-publisher/bin/tester/../../node_modules/dtslint/bin/index.js --noLint
	content-type-tests.ts(14,25): error TS2345: Argument of type 'Response' is not assignable to parameter of type '{ getHeader(key: string): string; }'.
  Types of property 'getHeader' are incompatible.
    Type '(name: string) => string | number | string[]' is not assignable to type '(key: string) => string'.
      Type 'string | number | string[]' is not assignable to type 'string'.
        Type 'number' is not assignable to type 'string'.
```

content-type just uses a type literal to cut out the part of `Response` that it cares about. Either it should update its type or change to requiring the full `Response` type, which it would need to import.